tools/build/scripts/utils/addSourceDir.js

```diff
@@ -17,15 +17,15 @@ if (process.env.RUSHSTACK_FILE_ERROR_BASE_FOLDER) {
   rootPackageJsonPath = path.resolve("../../../../package.json");
 }
 
-const rootPackageJson = require(rootPackageJsonPath);
+// Check if path to root package.json is valid.
+require(rootPackageJsonPath);
```

Comment:
does it make sense to maybe use `require.resolve`? 
https://nodejs.org/api/modules.html#requireresolverequest-options?

Reply:
Changed to use `require.resolve` instead of `path.resolve`